app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java

```diff
@@ -1067,5 +1069,49 @@ protected void onPostExecute(Void result) {
                 SnackBarHandler.show(parent, getString(R.string.error_on_tumblr));
         }
     }
+    public File saveBitmapToFile(File file){
```

Comment:
@vinaysajjanapu Yes, my intention over here is to compress the image in the sharing Activity. Since the size of the image is fairly big, it is taking a lot of time to share the image on Pinterest, Twitter and more which does not compress the image. To make uploading process fast I have compressed the image in this function. Thanks!